api/app/Http/Controllers/ArticleMetaController.php

```diff
@@ -9,16 +9,30 @@
 namespace App\Http\Controllers;
 
 use App\Http\Transformers\EloquentModelTransformer;
-use App\Repositories\ArticleRepository;
+use App\Models\Article;
+use App\Models\ArticleMeta;
+use Spira\Repository\Model\BaseModel;
 
 class ArticleMetaController extends ChildEntityController
 {
-    protected $validateRequestRule = 'required|string';
+    protected $validateParentRequestRule = 'required|string';
     protected $validateChildRequestRule = 'required|string';
```

Comment:
Can you rename this property to `$validateParentIdRule` and `$validateChildIdRule` as `Request` is misleading